### DISCUSSION TITLE
eSIM with srsRAN and open5gs

### DISCUSSION THREAD

**Discussion by AzeezEbrahim** [srsran/srsRAN_Project#323] [General Help] [answered]
I'm working with a USRP B210, srsRAN, and Open5GS core, using  [sysmoISIM-SJA2](https://sysmocom.de/products/sim/sysmousim/index.html)  cards. 

I'm interested in integrating eSIM capabilities. 

I noticed Sysmocom mentions eSIM support with their eUICC product [sysmocom eUICC for eSIM](https://sysmocom.de/products/sim/sysmocom-euicc/index.html) 

**Can I use this setup with eSIM?**

**Comment by robertfalkenberg** [comment, marked as answer]
The srsRAN project, i.e. the gNB, is independent of the type of SIM card.

However on that topic, I recommend [this talk](https://media.ccc.de/v/camp2023-57190-demystifying_esim_technology).

**Comment by WaledEzzi** [comment]
@AzeezEbrahim hope that you are doing well,,
kindly have you managed with the integrating eSIM capabilities.

as i am looking for any project as starting point for this topic.
